include/ringmesh/basic/frame.h

```diff
@@ -147,11 +147,79 @@ namespace RINGMesh
             return origin_;
         }
 
+        vecn< DIMENSION > coords_to_local(
```

Comment:
Why there is no implementation here?

Reply:
Because it implies a matrix inversion, and i didn't code yet (was waiting for other changes and compilation to code it using the generic inversion of matrices in geogram)